client/src/components/geoJSONMap.tsx

```diff
@@ -1,99 +1,113 @@
-import React, { useEffect, useState } from "react";
-import { MapContainer, GeoJSON, useMap } from "react-leaflet";
-import { GeoJsonObject, Feature, Geometry } from "geojson";
-import "leaflet/dist/leaflet.css";
-import L from "leaflet";
-import "./geoJSONMap.css";
+/**-------***-------***-------***-------***-------

```

Comment:
i believe this is one commit behind the develop branch right now. update this to the latest branch and then it should be good.